### DISCUSSION TITLE
How do you specify PFX password when using Spn login with pfx certificate?

### DISCUSSION THREAD

**Discussion by kkizer** [Azure/kubelogin#118] [Q&A]
I would like to use a [PFX certificate when using a service principal to login](https://github.com/Azure/kubelogin#spn-login-with-pfx-certificate) when using the exec plugin. The path to the PFX file is specified with --client-certificate. How do you specify the password on the PFX file? The kubelogin get-token --help doesn't show an argument like --client-certificate-password. Is --password overloaded so that it is also used with PFX certs? If you can't have a password on the PFX, I'll need to create a PFX without a password.

**Comment by weinong** [comment]
looks like it currently hard codes to empty password
https://github.com/Azure/kubelogin/blob/0fc3b3385f6431fd28b3ba806d48f857a1089054/pkg/token/serviceprincipaltoken.go#L85

so, to unblock yourself, do a cert conversion to strip off the password :D
